src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/MovementsResourceIntTest.kt

```diff
@@ -539,34 +539,21 @@ class MovementsResourceIntTest(
             }
           }
         }
-      }
 
-      // Corrupt the data copied during the merge in the same way as NOMIS does
-
-      // the type and subtype are null after copying from the merged booking
-      jdbcTemplate.update(
-        """
-        update OFFENDER_MOVEMENT_APPS set 
-          TAP_ABS_TYPE=null,
-          TAP_ABS_SUBTYPE=null
-        where OFFENDER_MOVEMENT_APP_ID = ${application.movementApplicationId}
-        """.trimIndent(),
-      )
```

Comment:
Dropped this merge related corruption because it was causing problems when saving the transaction with jdbc, couldn't get an update query to work with entityManager due to the JoinFormula, and it doesn't really matter anyway as it's not the interesting part of the corrupted data.